source/agora/consensus/Quorum.d

```diff
@@ -115,7 +115,10 @@ public QuorumConfig buildQuorumConfig (in PublicKey key,
         added[idx] = true;  // mark used
     }
 
-    quorum.nodes.sort;
+    () @trusted {
+        scope (failure) assert(0);
+        quorum.nodes.sort;
+    }();
```

Comment:
This looks unrelated ?

Reply:
I see why. You get the error message because, while sorting, `opCmp` is called, and unfortunately it is not annotated. Will fix.

Reply:
For reference:
https://github.com/bpfkorea/crypto/blob/374d775327cb381a5037deb88edb7e352636de2f/source/agora/crypto/ECC.d#L382-L392
Calls https://github.com/Geod24/bitblob/blob/5418160fe7751eaa1ec99870ae7c62bd1f9c58e7/source/geod24/bitblob.d#L237-L251 but the later doesn't require annotations because it's templated.

Reply:
Depends on https://github.com/bpfkorea/crypto/pull/19

Reply:
BTW crypto submodule update has been merged.
Commit cbb4169a3a23092d18463b8723a68680f36c5010  includes the changes to attributes.

Reply:
Will be included now if you rebase on upstream.

Reply:
OK, I will apply it. Thanks.

Reply:
Applied new `crypto` submodule.